lib/src/browser.dart

```diff
@@ -43,26 +42,17 @@ class Browser {
   final Function _matchesNavigator;
   final Function _parseVersion;
 
-  Version _version;
+  Version? _version;
 
-  Version get version {
-    if (_version == null) {
-      if (_parseVersion != null) {
-        _version = _parseVersion(Browser.navigator);
-      } else {
-        _version = Version(0, 0, 0);
-      }
-    }
-
-    return _version;
-  }
+  Version get version =>
+      _version ??= _parseVersion(Browser.navigator ?? TestNavigator());
 
   static List<Browser> _knownBrowsers = [
-    chrome,
+    internetExplorer,
```

Comment:
changed the order here so it's possible to use Chrome devtools to set an IE useragent and have platform_detect detect it as IE